cmd/stringer/golden_test.go

```diff
@@ -296,11 +327,222 @@ func (i Token) String() string {
 }
 `
 
+// Simple test: enumeration of type int starting at 0.
+const month_in = `type Month int
```

Comment:
warning: don't use underscores in Go names; const month_in should be monthIn (golint)